test/fixtures.ts

```diff
@@ -364,12 +370,8 @@ export const newWorkspaceUser = (params?: {
     key: randomDataGenerator.string({ length: 32 }),
     workspaceId: params?.workspaceId || v4(),
     spaceLimit: BigInt(spaceLimit),
-    driveUsage: BigInt(
-      randomDataGenerator.natural({ min: 1, max: spaceLimit }),
-    ),
-    backupsUsage: BigInt(
-      randomDataGenerator.natural({ min: 1, max: spaceLimit }),
-    ),
+    driveUsage: BigInt(0),
+    backupsUsage: BigInt(0),
```

Comment:
Adding a random value to these two fields looked as a good idea when I created this fixture, but it makes testing space related functions somehow tedious. So it is better to set then to 0 and just add usage when you need it!